src/tests/system/tests/test_ldap.py

```diff
@@ -144,3 +144,33 @@ def test_ldap__change_password_wrong_current(client: Client, ldap: LDAP, modify_
     client.sssd.start()
 
     assert not client.auth.passwd.password("user1", "wrong123", "Newpass123"), "Password change did not fail"
+@pytest.mark.topology(KnownTopology.LDAP)
```

Comment:
Hi,

please add two empty lines before starting a new test to follow python's coding style.

bye,
Sumit

Reply:
Fixed.